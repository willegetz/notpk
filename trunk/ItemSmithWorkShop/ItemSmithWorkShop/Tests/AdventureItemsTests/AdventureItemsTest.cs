using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using AdventureItems;
using ApprovalTests;

namespace AdventureItemsTests
{
	[TestClass]
	public class AdventureItemsTest
	{
		[TestMethod]
		public void TestCreateWeapon()
		{
			AdventureItemShop weaponShop = new WeaponItemSmith();
			AdventureItem weapon = weaponShop.OrderItem("Dagger");
			Approvals.Approve(weapon.GetItem());
		}
	}
}
