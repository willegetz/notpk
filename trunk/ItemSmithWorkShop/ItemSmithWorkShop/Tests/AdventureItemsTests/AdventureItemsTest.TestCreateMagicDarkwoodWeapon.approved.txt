+1 Darkwood Dagger:	'2312 gp'
Weight: '0.5 pound(s)'
To Hit: '+1'
Damage: '1d4 +1'
	The dagger is a common secondary weapon. You get a +2 bonus on
	Sleight of Hand checks made to conceal a dagger on your body.
	Darkwood weapons are always considered to be masterwork quality.